### DISCUSSION TITLE
Domain Renewal

### DISCUSSION THREAD

**Discussion by bayshaffer** [vercel/community#6200] [Help] [answered]
### Summary

One of my domains, `bayshaffer.com`, expired. I need it renewed if possible. I tried creating a case in hopes that it isn't too late to pay and was semi-redirected here. I'll upgrade my account, subscribe to [v0](https://v0.dev/), buy additional domains, really just the absolute most of whatever I have to do. I've updated my card on file and I authorize Vercel to charge it again. If I am too late, a step-by-step on how to begin the process of recovering the domain would be a great resource as I have never.

### Steps to Reproduce

Go to [Domains - Vercel ~ bayshaffer.com](https://vercel.com/bay/~/domains/bayshaffer.com) and you will see the issue described.

**Comment by swarnava** [comment, marked as answer]
Hi,

Can you create a Domain Renewal ticket from vercel.com/help ? Thanks!

**Reply by bayshaffer** [reply]
#00212078

**Reply by amyegan** [reply]
It looks like your domain was renewed with the help of one of our Customer Success Engineers today. Please follow up with your support case if you still have any trouble with it 🙏
